### PR TITLE
Fix release ip

### PR DESCRIPTION
**Pull request by Mmduh-483**
After gain IPAM's ip in CMD ADD release it if error happens

### CODE REVIEW

File: sriov/main.go [hunk 1]
```diff
@@ -79,6 +90,12 @@ func cmdAdd(args *skel.CmdArgs) error {
 		return errors.New("IPAM plugin returned missing IPv4 config")
 	}
 
+	defer func() {
+		if err != nil {
+			ipam.ExecDel(n.IPAM.Type, args.StdinData)
+		}
+	}()
+
```
Comment (ahalim-intel) [inline sriov/main.go]: Should we not add this block after L:101 below?

**Comment by zshi-redhat** [comment]
This looks like a series of commits fixing different issues, would you mind break them down to separate ones?

**Comment by Mmduh-483** [comment]
I will fix that

**Comment by Mmduh-483** [comment]
I will do another pull request with no dependencies

**Pull request closed without merging**